config/routes.rb

```diff
@@ -12,9 +16,10 @@
       end
       resources :manifests, only: [:create]
       resources :method_codes, only: [:index]
-      patch 'manifests', to: 'manifests#update'
-      get 'manifest', to: 'manifests#show'
+      patch 'manifests/:id', to: 'manifests#update'
```

Comment:
I think that if we added `:update` to line 17 that would be the same as doing this.